src/test/java/uk/gov/hmcts/reform/sscs/personalisation/WithRepresentativePersonalisationTest.java

```diff
@@ -61,9 +69,10 @@ public void givenSscsCaseData_shouldSetRepresentativeNameIfPresent(
             .build();
         return new Object[]{
             new Object[]{sscsCaseDataWithReps, "Manish Sharma"},
+            new Object[]{sscsCaseDataWithRepsOrgOnly, "Sir / Madam"},
```

Comment:
Use the uk.gov.hmcts.reform.sscs.config.AppConstants#REP_SALUTATION constant.

Reply:
Done